### ISSUE TITLE
Dutch NER

### ISSUE THREAD

**Issue by Niekvdplas** [amaiya/ktrain#82]
Hello Amaiya,

I don't know if you have trained the models for NER yourself, but if you have is it possible for you(or me, with some explanation on how to do it) to add a Dutch model? I have a [dataset ](https://www.clips.uantwerpen.be/conll2002/ner/) where the training can be done on.  I would like to help if it is much work :-)

**Comment by amaiya** [comment]
Yes, the ready-to-use NER models for English, Chinese , and Russian were trained by us.  A Dutch model should be easy to add, since the dataset your provided seems to be in an easy-to-ingest format.

**Comment by Niekvdplas** [comment]
Awesome! Looking forward to it. I hope you find some time soon :-)

**Comment by amaiya** [comment]
Hi @Niekvdplas The newest version of **ktrain** allows users to specify the URL to [pretrained fasttext word vectors](https://fasttext.cc/docs/en/crawl-vectors.html) (which supports 157 different languages).  I trained a Dutch NER model using the dataset you provided using pretrained Dutch word vectors and a cosine annealing learning rate schedule.  This achieved a test F1 score of ~74, which looks to be higher than Spacy's Dutch NER model (but lower than other languages like Chinese and English):
```python
import ktrain
from ktrain import text as txt
TDATA = 'data/dutch_ner/ned.train'
VDATA = 'data/dutch_ner/ned.testb'
WV_URL='https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.nl.300.vec.gz'
(trn, val, preproc) = txt.entities_from_conll2003(TDATA, val_filepath=VDATA,
                                                           wv_path_or_url=WV_URL)
model = txt.sequence_tagger('bilstm', preproc)
learner = ktrain.get_learner(model, train_data=trn, val_data=val, batch_size=128)
learner.fit(0.01, 1, cycle_len=5, checkpoint_folder='/tmp/dutch_checkpoints')
learner.validate(class_names=preproc.get_classes())
```
For now, since it is very easy to train an NER model in **ktrain** for any language given a dataset, I didn't include a trained Dutch model within the ktrain package, as I did for English and Chinese.  Later, we may move all the ready-to-use models to S3 in which case we can add Dutch in addition to others like spanish.

I've also added a `predictor_path` argument to `shallownlp.NER` so that it can be used for user-trained models, if desired.  Here, is the code to use a user-trained Dutch NER model with `shallownlp.NER` interface:

```python
predictor = ktrain.get_predictor(learner.model, preproc)
predictor.save('/tmp/my_dutch_nermodel')
ner = txt.shallownlp.NER('nl', predictor_path='/tmp/my_dutch_nermodel')
ner.predict("Marke Rutte is een Nederlandse politicus die momenteel premier van Nederland is.",
           merge_tokens=True)

# output
# [('Marke Rutte', 'PER'), ('Nederlandse', 'MISC'), ('Nederland', 'LOC')]
```

Finally, there is now an [example notebook for Dutch NER](https://github.com/amaiya/ktrain/blob/develop/examples/text/CoNLL2002_Dutch-BiLSTM.ipynb).
